lib/AliasFieldPlugin.js

```diff
@@ -41,11 +40,11 @@ AliasFieldPlugin.prototype.apply = function(resolver) {
 			path: request.descriptionFileRoot,
 			request: data
 		});
-		resolver.doResolve(target, obj, "aliased from description file " + request.descriptionFilePath + " with mapping '" + innerRequest + "' to '" + data + "'", createInnerCallback(function(err, result) {
+		resolver.doResolveWithErrorLogging(target, obj, "aliased from description file " + request.descriptionFilePath + " with mapping '" + innerRequest + "' to '" + data + "'", function(err, result) {
```

Comment:
I'm trying to understand what's to be changed here. From the sound of it `.resolve` is the only function directly called via the build, then all the `.doResolve` calls are made from Resolver plugins... is that right?

If so it would seem to me the only thing that this PR needs to change is `.resolve` around line 74 to be something like...
```javascript
return this.doResolve("resolve", obj, message, function(err, result) {
	if (err) {
		return this.doResolve("resolve", obj, message, createInnerCallback(onResolved, {
			log: writeLog,
			missing: missing,
			stack: callback.stack
		}, null));
	}

	return onResolved(err, result);
});
```